app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def full_title page_title = "", base_title = ""
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
so sánh blank?, empty?, present?, nil?

Reply:
Dùng blank? tốt hơn anh nhỉ. Phương thức blank? của Rails xử lý cả khoảng trắng, trong trường hợp truyền giá trị page_title toàn khoảng trắng nếu để empty? sẽ nhận kết quả | #{base_title}.